buildSrc/subprojects/ide/src/main/kotlin/org/gradle/gradlebuild/ide/IdePlugin.kt

```diff
@@ -88,6 +88,20 @@ open class IdePlugin : Plugin<Project> {
                         }
                     })
                 }
+                jdt {
+                    sourceCompatibility = JavaVersion.VERSION_1_8
+                    targetCompatibility = JavaVersion.VERSION_1_8
+                    javaRuntimeName = "JavaSE-1.8"
+/*
+FIXME: No idea why Kotlin does not compile the lines below
```

Comment:
What's the error message?

Reply:
with the code below
```
jdt {
     sourceCompatibility = JavaVersion.VERSION_1_8
     targetCompatibility = JavaVersion.VERSION_1_8
     javaRuntimeName = "JavaSE-1.8"
				
     file {
          withProperties { Properties properties ->        // <== line 97
         // Eclipse's view of projects treat circular dependencies as errors by default
         properties["org.eclipse.jdt.core.circularClasspath"] = "warning"
     }
}
```
Kotlin states that
```
C:\Users\fandre\Documents\git\gradle>gradlew build
Parallel execution is an incubating feature.

> Task :buildSrc:ide:ktlintMainCheck FAILED
C:\Users\fandre\Documents\git\gradle\buildSrc\subprojects\ide\src\main\kotlin\org\gradle\gradlebuild\ide\IdePlugin.kt:97:63: Not a valid Kotlin file (expecting an element)
```

Reply:
Signed-off

Reply:
Looks like there are 4 commits now and you only signed off one of them. Please rebase on `master` and sign both your original commits.